apps/admin-portal/src/components/roles/create-role-wizard/role-wizard.tsx

```diff
@@ -51,8 +51,8 @@ interface CreateRoleProps {
  */
 enum WizardStepsFormTypes {
     BASIC_DETAILS = "BasicDetails",
-    PERM_LIST = "PermissionList",
-    USER_LIST = "RoleUserList",
+    PERM_LIST= "PermissionList",
+    USER_LIST= "RoleUserList",
```

Comment:
```suggestion
    PERM_LIST = "PermissionList",
    USER_LIST = "RoleUserList",
```

Why is this space removed ?

Reply:
fixed with commit 5fb1030